### DISCUSSION TITLE
Blank screen with eminence skin

### DISCUSSION THREAD

**Discussion by Basche14** [SwagOtaku/plugin.video.kaito#40] [Q&A] [answered]
Hi, using skin eminence on Android tv, and going into the addon there is nothing showing. It has installed properly without issue, but I get a blank screen as I enter the addon.

**Comment by unkn0wn234** [comment, marked as answer]
> Hi, using skin eminence on Android tv, and going into the addon there is nothing showing. It has installed properly without issue, but I get a blank screen as I enter the addon.

 Change the view type around to addons or tv shows

**Comment by icpryde** [comment]
go into settings change view type to "addons" or "tv shows".

Then restart kodi and it should work now, sometimes you might need to do a cache clear and restart if it still didn't work.
